### DISCUSSION TITLE
Wrapper for bike sharing apps

### DISCUSSION THREAD

**Discussion by kribsi** [ubahnverleih/WoBike#284] [General]
It would be great to have a unified e scooter booking app that maintains accounts for all providers in the background and lets you book through a single interface, hiding the actual provider.
From what I see in the API docs, none of them allow bookings via api, or am I missing something? That would be too bad. 

Thx for any help

**Comment by ubahnverleih** [comment]
As a booking process is much more complex than just getting locations of vehicles and requires lot more test, accounts and a lot of payed bookings to test the flow, we decided to focus on scooter locations in this repo. It's even hard to keep up on this with API changes.

**Comment by BastelPichi** [comment]
There is one or two apps out there that do that, while taking a 10%-15% cut. (EScoot). With most providers renting isnt too hard. You can take a look at https://github.com/BastelPichi/pyscooter, which has an implementation of how to rent bolt scooters.
